doc/configuration.rst

```diff
@@ -2434,7 +2439,8 @@ captured. For several examples, see :ref:`capture_repr_examples`.
 
 To capture only data directed to standard output, configure ``'capture_repr'``
 to be an empty tuple: ``'capture_repr': ()``. This will imitate the behaviour
-of Sphinx-Gallery prior to v0.5.0.
+of Sphinx-Gallery prior to v0.5.0. Note however that this behaviour is not
+consistent with how Jupyter notebooks capture output by default!
```

Comment:
I think it is not necessarily setting `capture_repr` to empty tuple in general that we want to warn about, I think it's more altering it for specific files or code blocks when you've set it to something else globally.

Reply:
Hey @lucyleeow thanks for the review. I think you and @larsoner obviously have much more experience with this framework and with how users interact with it so please feel free to push the changes you see fit 👍🏽

My 2 cents is that both warnings seem to be redundant. The users will only use these settings knowing what they do so it should be obvious that setting a different value at the block-level will result in different or inconsistent behaviour to other blocks that inherit the global setting. i.e., I don't really see what new information the user is learning but warning them that changing the setting at the block-level will result in a different behaviour when compared to other blocks.. that's the whole point of block-level settings 😄

But again, I don't have strong feeling about this, so feel free to push or let me know what you'd like to see here instead.

Thanks in advance!!

Reply:
> The users will only use these settings knowing what they do so it should be obvious that setting a different value at the block-level will result in different or inconsistent behaviour to other blocks that inherit the global setting. i.e., I don't really see what new information the user is learning by warning them that changing the setting at the block-level will result in a different behaviour when compared to other blocks.. that's the whole point of block-level settings

@tpvasconcelos when thinking about this issue it's important to be clear about who "users" are. I think you are using the term to mean "consumers of sphinx-gallery, i.e., (largely) Python library authors building a gallery". My warning has more to do with "end users of a given Python library who are browsing and using the Sphinx-gallery built doc". It's for these latter (end-)users where the inconsistency or surprising behavior can happen. Hopefully the warning I added makes this a bit clearer!

Reply:
@larsoner gotcha! Sounds reasonable to me 👍🏽